.ci/scripts/get-latest-beats-sha.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BEATS_BRANCH=$(grep "BEATS_VERSION?=" Makefile | awk 'awk BEGIN { FS = "=" } ; { print $2 }')
+
+curl "https://api.github.com/repos/elastic/beats/commits?sha=${BEATS_BRANCH}" | jq --raw-output '.[0].sha' | xargs | cut -c 1-12
```

Comment:
I tried to use something like that  with `updatecli` since it support `kind: json`

```
  sha:
    name: Get commit
    kind: json
    spec:
      file: 'https://api.github.com/repos/elastic/ecs-logging/commits?path=spec%2Fspec.json&page=1&per_page=1'
      key: ".[*].[*].sha"
```

So I could get the latest given commit for a particular file, but I failed with the `key` search. I wonder whether you had the same kind of issue?

Reply:
Ah, cool.. I didn't try this one.. will try it now

Reply:
This worked!

https://github.com/elastic/apm-server/pull/10179/commits/039aea49cbd5c6abcc50298e6557e15212e33a04

Reply:
and https://github.com/elastic/apm-server/pull/10179/commits/a01a2fd6825d93c39b07392e7db5dff1f8cd1dfd to account for the branch name